readme.md

```diff
@@ -116,6 +116,20 @@ normalizeUrl('http://sindresorhus.com/', {removeTrailingSlash: false});
 //=> 'http://sindresorhus.com'
 ```
 
+##### removeDirectoryIndex
+
+Type: `boolean|Array<RegExp|String>`<br>
+Default: `false`
+
+Remove default directory index files from path that matches any of the provided strings or regexes. When
+the parameter is `true` the regex `/^index/` is used.
+
+```js
+normalizeUrl('www.sindresorhus.com/foo/index.html', {
+	removeDirectoryIndex: [/^index/]
```

Comment:
This example is the same as `true`. Show something that would actually make the regex option useful.